src/jit/lir.cpp

```diff
@@ -1029,7 +1157,7 @@ LIR::Range LIR::AsRange(GenTree* firstNode, GenTree* lastNode)
 }
 
 //------------------------------------------------------------------------
-// LIR::SetTreeSeq: Given a newly created, unsequenced HIR tree, sequence
+// LIR::SeqTree: Given a newly created, unsequenced HIR tree, sequence
```

Comment:
Please update the comment to reflect the fact that it eval-orders the tree as well.